homework1_portfolio_styles.css

```diff
@@ -0,0 +1,75 @@
+body {
+	font-family: helvetica;
+}
+
+nav {
+	font-size: 28px;
+	padding-top: 10px;
+	text-align: center;
+
+}
+
+h1 {
+	font-size: 40px;
+	text-align: center;
+}
+
+h2 {
+	font-size: 26px;
+	padding-left: 100px;
+}
+
+h4 {
+	font-size: 18px;
+	padding-left: 100px;
+}
+
+img { 
+	float: left;
+	margin-left: 100px;
+	margin-right: 15px;
+	margin-bottom: 10px;
+	margin-top: 10px;
+}
+
+div.secondblock {
```

Comment:
I see that your .secondblock, .thirdblock, and .fourthblock classes all have the same rules.  You can use the same class on multiple HTML tags.  Try simplifying this by just having 1 class declaration here and using that same class on all of these divs in your HTML.